tools/ci_build/github/azure-pipelines/templates/windows-build-tools-setup-steps.yml

```diff
@@ -49,7 +49,7 @@ steps:
       displayName: 'Install conda modules'
       inputs:
         filename: '$(Build.BinariesDirectory)\packages\python\scripts\conda.exe'
-        arguments: 'install -q --insecure -y pyopenssl setuptools wheel numpy'
+        arguments: 'install -q --insecure -y pyopenssl setuptools wheel numpy mysql-connector-python'
```

Comment:
This will install **mysql-connector** for all CI jobs. Since it depends on libprotobuf (with .h files), there's a possibility of impacting regular builds. 

Since **post-merge-jobs.yml**  is executing "templates/windows-build-and-test-steps.yml" , which is  separate build step, can we install this python package only when running code-coverage, and not for all CI runs?